app/jobs/globus_setup_job.rb

```diff
@@ -36,7 +36,10 @@ def globus_user_exists?(user_id)
   def create_globus_endpoint(work_version)
     user = work_version.work.owner
     # e.g. 'mjgiarlo/work1234/version1'
-    endpoint_path = "#{user.sunetid}/work#{work_version.work.id}/version#{work_version.version}"
+    endpoint_path = format(WorkVersion::GLOBUS_ENDPOINT_TEMPLATE,
+                           user_id: user.sunetid,
+                           work_id: work_version.work.id,
+                           work_version: work_version.version)
```

Comment:
As above, moving work version-specific logic out of jobs and into the `WorkVersion` model.